build/common.props

```diff
@@ -29,6 +29,8 @@
     <NetStandardImplicitPackageVersion>$(NetStandardVersion)</NetStandardImplicitPackageVersion>
     <LangVersion>13</LangVersion>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <!-- Suprpess the "Microsoft.Bcl.Memory 9.0.0 doesn't support net6.0 and has not been tested with it" warning -->
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
```

Comment:
Does this warning appear in all projects and for all target frameworks?
If not, I'd like us to have a more targeted suppression.

(in the affected csproj, and in a PropertyGroup guarded by a condition on the framewor,